lib/workers/repository/update/pr/body/index.ts

```diff
@@ -66,6 +73,14 @@ function massageUpdateMetadata(config: BranchConfig): void {
   });
 }
 
+function getSourceRootPath(sourceUrl: string | undefined): string {
+  if (sourceUrl?.startsWith('https://bitbucket.org/')) {
```

Comment:
we probably should detect the target platform here instead when fetching changelogs, so we don't have to do it twice.

Reply:
refactored to use common detectPlatform util, the same that release-notes (and many other parts) use.

Reply:
I like the idea of detecting the platform and parsing that around the update object, but that's a much bigger change/impact across the codebase than I'd like to take on under this ticket